.cirrus.yml

```diff
@@ -66,7 +71,7 @@ lint_task:
     folder: $HOME/.cargo/registry
   clippy_script:
     - . $HOME/.cargo/env
-    - cargo +$VERSION clippy --all-targets -- -D warnings
+    - cargo +$VERSION clippy --all-targets --all-features -- -D warnings
```

Comment:
We should have a run that doesn't build with `--all-features` to make sure we don't break building with a subset of the features. We don't need to test every variant, but at least testing the "minimal" and "I want all the things" variants would be nice!